website/content/docs/add-to-your-site.md

```diff
@@ -61,13 +61,13 @@ The second file, `admin/config.yml`, is the heart of your Netlify CMS installati
 You can also use Netlify CMS as an npm module. Wherever you import Netlify CMS, it automatically runs, taking over the current page. Make sure the script that imports it only runs on your CMS page. First install the package and save it to your project:
 
 ```bash
-npm install netlify-cms --save
+npm install netlify-cms-app --save
 ```
 
 Then import it (assuming your project has tooling for imports):
 
 ```js
-import CMS from 'netlify-cms'
+import CMS from 'netlify-cms-app'
```

Comment:
I think you'll need to explicitly call `CMS.init()` when using `netlify-cms-app`. Can you please verify it?

Reply:
I just tested this in a React project and you are correct, `CMS.init()` should be called before anything else.